package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cookie-dough",
-  "version": "0.1.0",
+  "version": "0.2.0-beta.0",
```

Comment:
⚠️ this should probably not be merged into master.

Reply:
Agree.  I'm actually inclined to do away with cookie-dough altogether, TBH.

Reply:
If you're interested, it turns out that `js-cookie` can be used both client-side and server-side.